src/styles/utils/_mixins.scss

```diff
@@ -31,3 +31,27 @@
     @content;
   }
 }
+
+
+@mixin flex-item-center($direction){
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  flex-direction:$direction;
+}
+
+@mixin font-heading() {
```

Comment:
Mixins should be reusable and general. You can use parameters to mixins to make them reusable. The way it is right now is static and may conflict with other's requirements.

Reply:
Deleted those mixin which was not reusable.